pom.xml

```diff
@@ -42,6 +42,12 @@
 			<artifactId>h2</artifactId>
 			<scope>runtime</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/com.google.code.gson/gson -->
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>2.8.5</version>
+		</dependency>
```

Comment:
spring internally uses Jackson which has greater annotation support. Do you see any advantages of using gson over the spring boot's default jackson library? I am just asking. Please reply.

Reply:
There are so many methods in gson which are helpful. Earlier I used some methods to convert the complex list data and also retrieving some data from collections

Reply:
Ok. I have read through some article. jackson+spring evolved a lot. So, for week3 assignment use jackson. so you will have better understanding of both.